src/main/java/server/Router.java

```diff
@@ -40,30 +41,33 @@ private void addToRoutes(HTTPMethods method, String path, Handler handler) {
     }
 
     public Response generateResponse(Request request) {
+        String path = request.getRequestPath();
+        if(path.contains("?")){
+            path = path.split("\\?")[0];
```

Comment:
👍 to detecting query params! Do we do anything with them?

Reply:
Yes - splitting off the query params was necessary to match the request path with the defined endpoint in the router--so the query params are left in the `request` object that gets passed to `generateResponse` which then uses the query params to make the actual request to the api.